### ISSUE TITLE
Add Qristal to the list

### ISSUE THREAD

**Issue by patscott** [qosf/awesome-quantum-software#124]
Hi - it would be great if Qristal 

https://github.com/qbrilliance/qristal

could be added to the list.  Let us know if you would like us (Quantum Brilliance) to put up a PR to that effect, we'd be more than happy.

@jasonpillay1

**Comment by dlyongemallo** [comment]
Yes, please send a PR. Also FYI, I'm on vacation and away from computer until next week, so I won't be able to look at it until then. No rush.

**Comment by dlyongemallo** [comment]
Hi, @patscott. Did you send a PR?

**Comment by patscott** [comment]
Hi @dlyongemallo - thanks for the bump; I'm on parental leave until Dec, but @jasonpillay1 should be able to advise on the progress working up a PR.

**Comment by jasonpillay1** [comment]
Hi David,

Sorry for the delay. We’re pretty inundated at the moment, so we’ll send a PR as soon as we can.

Cheers,
Jason

**Comment by dlyongemallo** [comment]
Hi!

I had a look at the repository. Is the SDK itself open-source? I couldn't locate the actual source files and only saw links to docker images.

**Comment by jasonpillay1** [comment]
Hi, yes it is. Most of the source code is located in the qristal-core repo.

Cheers,
Jason

**Comment by dlyongemallo** [comment]
What's the status of this? Would like to finish this up if possible before winter holidays, if possible.

**Comment by patscott** [comment]
@jasonpillay1 and I are going to take care of this together in January, so expect a PR by the end of that month.

**Comment by dlyongemallo** [comment]
Hi, happy new year! I know everyone is very busy during the holidays and things can sometimes slip, so this is a gentle prod to send the PR in case you've forgotten.

**Comment by dlyongemallo** [comment]
I've added an entry for Qristal @patscott @jasonpillay1. Thanks for the suggestion!
